e2e/__tests__/confirm_API_data_browse.spec.ts

```diff
@@ -16,5 +18,14 @@ test('API Datasource Browsing', async ({ page }) => {
   await page.goto('/');
   const elements = await page.locator('[id="TestAPIDataSource"]'); // After first run there will be more than one element by this name
   await elements.nth(0).click(); // Only look for the first element
-  await expect(page.getByText('AUTHOR')).toBeVisible();
+  await expect(page.getByText('Author')).toBeVisible();
```

Comment:
Just out of interest, what's this line for?

Reply:
If the control is visible, the page must be loaded.